code/iaas/agent-instance/src/main/java/io/cattle/platform/agent/instance/dao/AgentInstanceDao.java

```diff
@@ -30,4 +30,5 @@
 
     void populateNicAndIp(NetworkServiceInfo service);
 
+    List<Long> getAgentProvider(String providedServiceLabel, Long accountId);
```

Comment:
Nit: if accountId is never meant to be null, can we change Long to long in the signature?